index.html

```diff
@@ -26,6 +26,165 @@ <h1 class="career">I'm a software developer</h1>
                 <li><a class="twitter" href="https://twitter.com/Juanse77930"><img src="images/twitter.svg" alt="twitter"></a></li>
             </ul>
         </section>
+        <section class="whole-works-section">
+            <section class="works-title">
+                <a class="works"><img src="Backgroud-images/Disabled.png" alt="works-button"></a>
+                <h2 class="recent-works">My Recent Works</h2>
+                <img class="separation-bar" src="Backgroud-images/Indicator.png" alt="separation-bar">
+            </section>
+            <section class="works-container">
+                <img class="image-grey" src="/Backgroud-images/Img_Plaholder.png" alt="image">
+                <div class="multi">
+                    <section class="multi-post">
+                        <h2 class="multi-post-title">Multi-Post Stories Gain+Glory</h2>
+                        <section class="buttons">
```

Comment:
- [ ] OPTIONAL: I have noticed that all the cards are not centred, for mobile screens  please kindly consider centering them 🤗
<img width="425" alt="Screenshot 2022-02-11 at 21 08 47" src="https://user-images.githubusercontent.com/77038610/153636585-ec3b54b8-3c91-4127-9ecc-49f3e4dc65c1.png">
 You can easily do it with `flexbox` property